modules/cluster-cloud-controller-config-osp.adoc

```diff
@@ -0,0 +1,328 @@
+//
+// * installing/installing_openstack/installing-openstack-cloud-config-reference.adoc
+
+:_content-type: REFERENCE
+[id="cluster-cloud-controller-config_{context}"]
+= The {rh-openstack} Cloud Controller Manager (CCM) config map
+
+A {rh-openstack-first} CCM config map defines how your cluster interacts with your {rh-openstack} cloud. By default, this configuration is stored under the `cloud.conf` key in the `cloud-conf` config map in the `openshift-cloud-controller-manager` namespace. 
+
+[IMPORTANT]
+====
+The `cloud-conf` config map is generated from the `cloud-provider-config` config map in the `openshift-config` namespace.
+
+To change the settings that are described by the `cloud-conf` config map, modify the `cloud-provider-config` config map. 
+
+As part of this synchronization, the CCM Operator overrides some options. For more information, see "The {rh-openstack} Cloud Controller Manager".
+====
+
+For example:
+
+.An example `cloud-conf` config map
+[source,yaml]
+----
+apiVersion: v1
+data:
+  cloud.conf: |
+    [Global] <1>
+    secret-name = openstack-credentials
+    secret-namespace = kube-system
+    region = regionOne
+    [LoadBalancer]
+    use-octavia = True
+kind: ConfigMap
+metadata:
+  creationTimestamp: "2022-12-20T17:01:08Z"
+  name: cloud-conf
+  namespace: openshift-cloud-controller-manager
+  resourceVersion: "2519"
+  uid: cbbeedaf-41ed-41c2-9f37-4885732d3677
+----
+<1> Set global options by using a `clouds.yaml` file rather than modifying the config map.
+
+The following options are present in the config map. Except when indicated otherwise, they are mandatory for clusters that run on {rh-openstack}. 
+
+// [id="ccm-config-global-options"]
+// == Global options
+
+// The following options are used for {rh-openstack} CCM authentication with the {rh-openstack} Identity service, also known as Keystone. They are similiar to the global options that you can set by using the `openstack` CLI.
+
+// |===
+// | Option | Description
+
+// | `ca-file`
+// | Optional. The CA certificate bundle file for communication with the {rh-openstack} Identity service. If you use the HTTPS protocol with The Identity service URL, this option is required.
+
+// | `cert-file`
+// | Optional. The client certificate path to use for client TLS authentication.
+
+// | `key-file`
+// | Optional. The client private key path to use for client TLS authentication.
+
+// | `region`
+// | The Identity service region name.
+
+// | `trust-id`
+// | The Identity service trust ID. A trust represents the authorization of a user, or trustor, to delegate roles to another user, or trustee. Optionally, a trust authorizes the trustee to impersonate the trustor. You can find available trusts by querying the `/v3/OS-TRUST/trusts` endpoint of the Identity service API.
+
+// | `trustee-id`
+// | The Identity service trustee user ID.
+
+// | `trustee-password`
+// | The Identity service trustee user password.
+
+// | `application-credential-id`
```

Comment:
The options application-credential-id and application-credential-secret were tested by QE and the user is allowed to set it. Could we mention it somewhere?

Reply:
This is kind of by accident. The second we start setting this in `clouds.yaml` there will be a conflict. I'm not actually sure what happens in this situation but I would rather push people to configure stuff via `clouds.yaml` exclusively where possible.

```yaml
clouds:
  foo:
    auth:
      auth_url: <auth-url>
      application_credential_id: "<credential-id>"
      application_credential_secret: "<credential-secret>"
```

wdyt?

Reply:
ah ok, we can leave it out then